src/job_cache/job_cache.cpp

```diff
@@ -219,16 +231,20 @@ FindJobResponse Cache::read(const FindJobRequest &find_request) {
     MessageParserState state = parser.read_messages(messages);
 
     if (state == MessageParserState::StopFail) {
-      // TODO: Try to reconnect to the daemon, launching our own if need be.
-      //       if that fails then depending on user preference either fail
-      //       or return a cache miss.
+      if (false) {
```

Comment:
I starred at this for so long before I realized this was dead code lol.

Reply:
yeah, this is where the config check for "cache miss instead of fail" goes

Reply:
In that case this logic should be moved outside of this function so that we can declare it a failure only once we've tried reconnecting multiple times.